changelog/13028.improvement.rst

```diff
@@ -0,0 +1,3 @@
+Remove (unreleased) 1-second join from threadexception plugin.
```

Comment:
Given the original change was not released yet, we can skip this changelog entry.